feature/networkinstance/feature.textproto

```diff
@@ -33,6 +33,17 @@ telemetry_path {
   path: "/network-instances/network-instance/state/type"
 }
 
+# Static routes
+config_path {
```

Comment:
Can we update the Config Library code as well.

Reply:
Done have added this to the config library code!

Reply:
Hmm I don't see the code

Reply:
Added and updated ni test as well